boards.txt

```diff
@@ -505,3 +507,57 @@ mdbt50qrx.menu.debug.l2.build.debug_flags=-DCFG_DEBUG=2
 mdbt50qrx.menu.debug.l3=Level 3 (Segger SystemView)
 mdbt50qrx.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3
 mdbt50qrx.menu.debug.l3.build.sysview_flags=-DCFG_SYSVIEW=1
+
+# ----------------------------------
+# Makerdiary nRF52840 MDK USB Dongle
+# ----------------------------------
+mdk52840.name=Makerdiary nRF52840 MDK Dongle
+
+# VID/PID for bootloader with/without UF2, Arduino + Circuitpython App
+mdk52840.vid.0=0x239A
```

Comment:
Confirmation: Is it correct?  (I cannot verify because I don't have this device.)